book/configuration.md

```diff
@@ -3,10 +3,16 @@
 ## Nushell Configuration with `env.nu` and `config.nu`
 
 Nushell uses a configuration system that loads+runs two Nushell script files at launch time:
-First, `env.nu`, then `config.nu`.
-Paths to these files can be found by calling `echo $nu.env-path` and `echo $nu.config-path`.
-`env.nu` is meant to define the environment variables which are then available within `config.nu`.
-`config.nu` can be used to add definitions, aliases, and more to the global namespace.
+
+ - `env.nu` is used to define environment variables.
```

Comment:
It's an important distinction here that these environmental variables are available within config.nu. This means that env.nu gets sourced first.